controllers/UsersController.js

```diff
@@ -0,0 +1,43 @@
+import status from '../helpers/StatusResponse';
+import models from '../models';
+
+
+const { users, profiles, roles } = models;
+
+/**
+* @description UsersController class
+*/
+class UsersController {
+  /**
+   * @description Fetch all the users
+   * @param {Object} req - HTTP Request
+   * @param {Object} res - HTTP Response
+   * @return {Object} Returned object
+   */
+  static async list(req, res) {
+    try {
+      const authorsList = await users.findAll({
+        include: [profiles, {
+          model: roles,
+          as: 'roles',
+          where: {
+            role: 'author'
+          }
+        }],
+        attributes: { exclude: ['password'] }
+      });
+      if (authorsList) {
```

Comment:
This is not grammatically correct. `authorList` or `authors` not `authorsList`. Also, if it results null or the array is empty, you are not handling for that

Reply:
Thanks for pointing this out, I will refactor immediately.